scripts/common.sh

```diff
@@ -33,7 +33,7 @@ metadata:
   name: ${csr_name}
 spec:
   signerName: "kubernetes.io/kube-apiserver-client"
-  request: "$(base64 "${csr_file}" | tr -d "\n\r")"
+  request: "$(cat "${csr_file}" | base64 | tr -d "\n\r")"
```

Comment:
super minor nit-pick... It's not necessary to use `cat` here
`base64 < "${csr_file}" | tr -d "\n\r"`
is slightly cleaner than
`cat "${csr_file}" | base64 | tr -d "\n\r"`